artifacts/openapi-spec/public.yaml

```diff
@@ -0,0 +1,562 @@
+                type: array
+                items:
+                  type: string
+              examples:
+                example-1:
+                  value:
+                    - string
+      operationId: get-solvers
+    put:
+      summary: Update solvers
+      operationId: put-solvers
+      responses:
+        '200':
+          description: OK
+      tags:
+        - solvers
+components:
+  schemas:
+    User_create:
+      title: User_create
+      x-stoplight:
+        id: uib7be9cxaivf
+      type: object
+      properties:
+        username:
+          type: string
+        password:
+          type: string
+        email:
+          type: string
+      required:
+        - username
+        - password
+        - email
+    User_info:
+      title: User_info
+      x-stoplight:
+        id: o990xy1y1f5i8
+      type: object
+      properties:
+        id:
+          type: string
+        username:
+          type: string
+        email:
+          type: string
+        jobIds:
+          type: array
+          items:
+            type: string
+        modelIds:
+          type: array
+          items:
+            type: string
+    Job_info:
+      title: Job_info
+      x-stoplight:
+        id: ln3dyvs5h60g1
+      type: object
+      properties:
+        id:
+          type: string
+        solverTypes:
+          type: array
+          items:
+            type: string
+        modelId:
+          type: string
+        dataId:
+          type: string
+        userId:
+          type: string
+      required:
+        - solverTypes
+        - modelId
+    Job_create:
+      title: Job_create
+      x-stoplight:
+        id: ov0zdtn4094u5
+      type: object
+      properties:
+        id:
+          type: string
+        solverTypes:
+          type: array
+          items:
+            type: string
+        modelId:
+          type: string
+        dataId:
+          type: string
+        options:
+          type: object
+          properties:
+            timeoutMs:
+              type: integer
+            maxMemoryMb:
+              type: integer
+            noOfVcpu:
+              type: integer
+    Job_result:
+      title: Job_result
+      x-stoplight:
+        id: dnvaj0bgb14wn
+      type: object
+      properties:
+        id:
+          type: string
+        solverType:
+          type: string
+        solution:
+          type: string
+        solveTimeMs:
+          type: integer
+    Model_info:
+      title: Model_info
+      x-stoplight:
+        id: txnq5kqs4wjas
+      type: object
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+        content:
+          type: string
+        userId:
+          type: string
+    Data_info:
+      title: Data_info
+      x-stoplight:
+        id: ilykln96rl0fs
+      type: object
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+        content:
+          type: string
```

Comment:
`data_info` skal måske have en reference til `model_id` for den model den leverer data for, bare for god ordens skyld?